src/Controller/UI/ReportDatasetSubmission.php

```diff
@@ -0,0 +1,160 @@
+<?php
```

Comment:
Should this also have a link on the admin page?

Reply:
We'll let the PO decide, was not in the ticket.